specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ManagedLedgerDigestUploads.json

```diff
@@ -0,0 +1,335 @@
+          }
+        },
+        "x-ms-examples": {
+          "Gets the current ledger digest upload configuration for a database.": {
+            "$ref": "./examples/ManagedLedgerDigestUploadsGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagedLedgerDigestUploads"
+        ],
+        "description": "Enables upload ledger digests to an Azure Storage account or an Azure Confidential Ledger instance.",
+        "operationId": "ManagedLedgerDigestUploads_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "ledgerDigestUploads",
+            "in": "path",
+            "description": "The name of the Ledger Digest Upload Configurations.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "current"
+            ],
+            "x-ms-enum": {
+              "name": "ManagedLedgerDigestUploadsName",
+              "modelAsString": true
+            }
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The Ledger Digest Storage Endpoint.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedLedgerDigestUploads"
+            }
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the ledger digest upload configuration",
+            "schema": {
+              "$ref": "#/definitions/ManagedLedgerDigestUploads"
```

Comment:
>ManagedLedgerDigestUploads [](http://example.com/codeflow?start=37&length=26)

Try naming Model as singular, something like ManagedLedgerDigestUpload or ManagedLedgerDigestUploadSetting or ManagedLedgerDigestUploadConfiguration etc. whichever is more appropriate.

Reply:
As mentioned above, we are advised to use plural names in elements in URL. So at other places in code as well, we use similar name. Please let me know if its fine